index.html

```diff
@@ -0,0 +1,213 @@
+              alt="Profile Pic"
+              />
+              <input type="text" placeholder="What's happening?" />
+            </form>
+            <!-- feed-icons -->
+            <ul class="eight-columns no-decoration">
+              <li>
+                <img class="feed-menu-icon" src="/icons/picture.png" alt="picture">
+              </li>
+              <li>
+                <img class="icon-small" src="/icons/gif.png" alt="gif">
+              </li>
+              <li>
+                <img class="icon-small" src="/icons/bar-graph.png" alt="graph">
+              </li>
+              <li>
+                <img class="icon-small" src="/icons/happiness.png" alt="smiley">
+              </li>
+              <li>
+                <img class="icon-small" src="/icons/calendar.png" alt="calendar">
+              </li>
+              <li></li>
+              <li></li>
+              <li>
+                <button class="post-tweet">Tweet</button>
+              </li>
+            </ul>
+          </div>
+          <!-- <div class="main-box"></div> -->
+          
+          <div class="responsive_rows post-box">
+          <div class="post__body">
+            <div class="post__header">
+              <div class="post__headerText">
+                
+                <span class="two-column-expand-space">
+                  <img
+                  src="https://img.search.brave.com/PU4XglJ1LbZ43RGtsowE0WKSw_VXOh2oHxgbQ2kwxfw/rs:fit:240:240:1/g:ce/aHR0cDovL2Zhcm0z/LnN0YXRpYy5mbGlj/a3IuY29tLzI3MTMv/NDA5ODI1OTc2OV82/OWFjODFkZWRhX20u/anBn"
+                  alt="Profile Pic"
+                  />
+                  M...
+                  <span class="post__header-Special"
+                    ><span class="material-icons post__badge"> verified </span
+                    >@Mubarak</span
+                  >
+                </span>
+              </div>
+              <div class="responsive-rows">
+                <div class="post">
+                  <div class="post__headerDescription">
+                    <p>1 - Amir Khan vs Kell Brook.</p>
+                    <a href="https://www.skysports.com/boxing"> Watch live</a>
+                  </div>
+                  <img
+                    src="https://img.search.brave.com/khlf_hjYm7Odq9bMccxhKQLM8IrcZC5b8BhDEKo3-c0/rs:fit:1200:940:1/g:ce/aHR0cHM6Ly9pMC53/cC5jb20vdmlyYWwu/b2xhc21lZGlhdHYu/Y29tL3dwLWNvbnRl/bnQvdXBsb2Fkcy8y/MDIyLzAyL1dBVENI/LUFtaXItS2hhbi12/cy1LZWxsLUJyb29r/LUZ1bGwtRmlnaHQt/U3RyZWFtaW5nLU9u/bGluZS5qcGc_em9v/bT0yJnJlc2l6ZT03/ODAlMkM0NzA"
+                    alt=""
+                  />
+                  <div class="post__footer">
+                    <span class="material-icons"> repeat </span>
+                    <span class="material-icons"> favorite_border </span>
+                    <span class="material-icons"> publish </span>
+                  </div>
+                </div>
```

Comment:
I don't see where the div class="post" closes? for me, in the tweets, there is one too few </div>

Reply:
Thanks your right I'll get that sorted.